src/cogs/admin_commands.py

```diff
@@ -126,3 +127,48 @@ async def create_channel_teams(self, ctx, channel_id, number_of_teams=2):
             )
 
         await ctx.channel.send(embed=teams_embed)
+
+    @commands.command(name="send-survey")
+    @commands.has_role(ADMIN_ROLE_ID)
+    async def send_survey(self, ctx, survey_id):
+        """
+        Only member with admin role can run this command
+        Command sends survey (id of survey in first parameter) to all users who are podcast-fans
```

Comment:
Here you mention "podcast-fans" but in the code the "alias" is "survey-fan". Let's use just one. IMHO `survey-fan` is better. `surveyable` might be even better though, but less "fun".

Reply:
podcast-fans was a mistake, it should be survey-fan in the comment as well as in the code